lazer/sdk/js/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@pythnetwork/pyth-lazer-sdk",
   "version": "4.0.0",
   "description": "Pyth Lazer SDK",
+  "engines": {
+    "node": ">=20"
```

Comment:
added this, per discussion over in our Slack thread about missing an engine and having bogus linting errors thrown